app/src/main/java/fr/free/nrw/commons/review/ReviewImageFragment.java

```diff
@@ -66,6 +66,8 @@ private String updateCategoriesQuestion() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        user = SAVED_USER;
```

Comment:
How does this help?

Reply:
Can you please delete this line @shabar-shab , so that we can review further:)